conftest.py

```diff
@@ -112,8 +112,8 @@ def generate_initial_config():
 BASE_DIR = ROOT_DIR
 DATA_DIR = fs.path.join(ROOT_DIR, 'colabTestData')
 # mscolab data directory
-MSCOLAB_DATA_DIR = fs.path.join(DATA_DIR, 'filedata')
-MSCOLAB_SSO_DIR = fs.path.join(DATA_DIR, 'datasso')
+FILE_DATA = fs.path.join(DATA_DIR, 'filedata')
```

Comment:
Is there some kind of naming convention to follow? Other directories have the _DIR suffix for example. Also, from the name it is unclear what this even contains. Maybe `OPERATIONS_DATA` might make more sense?